app/src/main/java/com/eveningoutpost/dexdrip/UtilityModels/Blukon.java

```diff
@@ -141,21 +140,9 @@ public static boolean checkBlukonPacket(byte[] buffer) {
             if (currentCommand.startsWith("810a00")) {//ACK sent
                 //ack received
 
-                if (JoH.pratelimit(BLUKON_GETSENSORAGE_TIMER, GET_SENSOR_AGE_DELAY)) {
```

Comment:
I haven't followed the data flow in this class, are you sure about these deleted lines?

Reply:
thanks for reviewing, yes I am quiet sure, this code portion can be Found at line 247.
no idea why this code porion was 2 times here, historical thing.